app/posts/smoking-cessation/2025/03/2025-03-20-smoking-cessation-pilot.md

```diff
@@ -0,0 +1,143 @@
+---
+title: "Smoking cessation pilot"
+description:
+  "Summary of work done to support the launch of a smoking cessation support service."
+date: 2025-03-20
+tags:
+---
+
+To achieve the government goal of a smoke-free Britain by 2030, populations with increasing dependence need access to smoking treatment services which meets their needs.
+
+Currently, people who want to stop smoking tobacco can use:
+- in-person NHSE services via routes like their pharmacy or GP
+- digital apps
+- work out a plan on their own
+
+People who want more guided support but cannot access in-person services for any reason do not have an NHSE digital service to use.
```

Comment:
Needs a bit more clarity? What does "NHSE digital service" actually represent? Or is it more "if you can't go in person there is often no alternative"?

Reply:
I guess my overall question would be... what's different? What makes an NHSE digital service different from

• in-person NHSE services via routes like their pharmacy or GP
• digital apps
• work out a plan on their own

?